src/v2/Apps/Artist/Routes/Consign/Components/__tests__/ArtistConsignMeta.jest.tsx

```diff
@@ -3,10 +3,6 @@ import { mount } from "enzyme"
 import React from "react"
 import { ArtistConsignMeta } from "../ArtistConsignMeta"
 
-jest.mock("v2/Utils/getENV", () => ({
-  getENV: () => "https://artsy.net",
-}))
```

Comment:
this was blowing up our tests, since we're now grabbing isomorphic values from the relay network middleware